instance/instance.go

```diff
@@ -260,10 +260,16 @@ func (s *Store) Update(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		if versionErr := s.AddVersionDetailsToInstance(ctx, currentInstance.InstanceID, datasetID, edition, instance.Version); versionErr != nil {
-			log.Event(ctx, "update instance: datastore.AddVersionDetailsToInstance returned an error", log.ERROR, log.Error(versionErr), editionLogData)
-			handleInstanceErr(ctx, versionErr, w, logData)
-			return
+		// update dp-graph instance node (only for non-cantabular types)
+		if currentInstance.Type == models.CantabularBlob.String() || currentInstance.Type == models.CantabularTable.String() {
+			editionLogData["instance_type"] = instance.Type
+			log.Event(ctx, "skipping dp-graph instance update because it is not required by instance type", log.INFO, editionLogData)
+		} else {
```

Comment:
Could this be an 'else if' ?

Reply:
I considered doing a `switch .. case` but as there are only 2 options, I decided to just do an if. But we might have other types in the future?
I think that currently cmd instances don't define a type, and didn't want to break anything else that was working, so I decided to keep the call to dp-graph by default.